cmd/root.go

```diff
@@ -22,42 +23,97 @@ var config configpkg.ConfigType
 var rootCmd = &cobra.Command{
 	Use:   "eirini-loggregator-bridge",
 	Short: "eirini-loggregator-bridge streams Eirini application logs to CloudFoundry loggregator",
+	PreRun: func(cmd *cobra.Command, args []string) {
+
+		viper.BindPFlag("operator-webhook-host", cmd.Flags().Lookup("operator-webhook-host"))
+		viper.BindPFlag("operator-webhook-port", cmd.Flags().Lookup("operator-webhook-port"))
+		viper.BindPFlag("operator-service-name", cmd.Flags().Lookup("operator-service-name"))
+		viper.BindPFlag("operator-webhook-namespace", cmd.Flags().Lookup("operator-webhook-namespace"))
+		viper.BindPFlag("register", cmd.Flags().Lookup("register"))
+		viper.BindPFlag("graceful-fail-time", cmd.Flags().Lookup("graceful-fail-time"))
+		viper.BindPFlag("graceful-success-time", cmd.Flags().Lookup("graceful-success-time"))
+		viper.BindPFlag("downloader-entrypoint", cmd.Flags().Lookup("downloader-entrypoint"))
+		viper.BindPFlag("executor-entrypoint", cmd.Flags().Lookup("executor-entrypoint"))
+		viper.BindPFlag("uploader-entrypoint", cmd.Flags().Lookup("uploader-entrypoint"))
+		viper.BindPFlag("opi-entrypoint", cmd.Flags().Lookup("opi-entrypoint"))
+
+	},
 	Run: func(cmd *cobra.Command, args []string) {
 		var err error
+		webhookHost := viper.GetString("operator-webhook-host")
+		webhookPort := viper.GetInt32("operator-webhook-port")
+		serviceName := viper.GetString("operator-service-name")
+		webhookNamespace := viper.GetString("operator-webhook-namespace")
+		register := viper.GetBool("register")
+		gracefulFailTime := viper.GetString("graceful-fail-time")
+		gracefulSuccessTime := viper.GetString("graceful-success-time")
+		downloaderEntrypoint := viper.GetString("downloader-entrypoint")
+		executorEntrypoint := viper.GetString("executor-entrypoint")
+		uploaderEntrypoint := viper.GetString("uploader-entrypoint")
+		opiEntrypoint := viper.GetString("opi-entrypoint")
 
 		LogDebug("Namespace: ", config.Namespace)
 		LogDebug("Loggregator-endpoint: ", config.LoggregatorEndpoint)
 		LogDebug("Loggregator-ca-path: ", config.LoggregatorCAPath)
 		LogDebug("Loggregator-cert-path: ", config.LoggregatorCertPath)
 		LogDebug("Loggregator-key-path: ", config.LoggregatorKeyPath)
+
+		LogDebug("Webhook listening on: ", webhookHost, webhookPort)
+		LogDebug("Webhook namespace: ", webhookNamespace)
+		LogDebug("Webhook serviceName: ", serviceName)
+		LogDebug("Webhook register: ", register)
+
 		LogDebug("Starting Loggregator")
+		if webhookHost == "" {
+			LogDebug("required flag 'operator-webhook-host' not set (env variable: OPERATOR_WEBHOOK_HOST)")
```

Comment:
good catch, this is a copy-paste error. It should be fatal as it's used anyhow - because if you don't start it with a service ref it is advertized at kube level, so it's used also while registering https://github.com/cloudfoundry-incubator/eirinix/blob/master/manager.go#L518

Reply:
Moved to warn level in https://github.com/cloudfoundry-incubator/eirini-loggregator-bridge/pull/26/commits/91bc05a605a4cdfc5dc65a6dd3f0d9de6891407d